include/sparrow/layout.hpp

```diff
@@ -203,19 +208,26 @@ namespace sparrow
     }
 
     template <class T>
-    auto primitive_layout<T>::element(size_type i) -> reference
+    auto primitive_layout<T>::value(size_type i) -> inner_reference
     {
         assert(i < size());
         return i[data()];
```

Comment:
please, no ;) `data()[i]` is far easier to read.

Reply:
+1